onnxruntime/core/optimizer/initializer.h

```diff
@@ -268,6 +322,22 @@ class Initializer final {
         }
         break;
       }
+      case ONNX_NAMESPACE::TensorProto_DataType_INT32: {
+        int32_t* dst = data<int32_t>();
+        const int32_t* src = other.data<int32_t>();
+        for (int i = 0; i < n; i++) {
+          dst[i] += src[i];
+        }
+        break;
+      }
+      case ONNX_NAMESPACE::TensorProto_DataType_INT64: {
```

Comment:
We should really just templatize the logic into a small helper function, but that can be done later.

Reply:
yeah I completely agree, just didn't want to introduce more changes right before the release.